### PR TITLE
chore: update issue templates

### PR DESCRIPTION
**Pull request by u11d-bartlomiej-galezowski**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: u11d-bartlomiej-galezowski, u11d-michal-miler
+
+---
+
+**Describe the bug**
```
Comment (u11d-michal-miler) [inline .github/ISSUE_TEMPLATE/bug_report.md:9]:
```suggestion
**Bug description**
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```
Comment (u11d-michal-miler) [inline .github/ISSUE_TEMPLATE/bug_report.md:12]:
```suggestion
**Steps to reproduce**

1. 
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+Steps to reproduce the behavior:
+1. 
+
+**Expected behavior**
```
Comment (u11d-michal-miler) [inline .github/ISSUE_TEMPLATE/bug_report.md:16]: 1. What do you think about additional `Current behavior` section?
2. I think we should check MD files by Prettier (empty line between lists or paragraph sections for instance)

```suggestion
**Current behavior**

A clear and concise description of current behavior.

**Expected behavior**

A clear and concise description of what you expected to happen.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Environemnt (please complete the following information):**
+ - Node: [e.g. node18.14.0]
```
Comment (u11d-michal-miler) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]:
```suggestion
 - Node: [e.g. 18.14.0]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+ - Serverless framework: [e.g. sls3.26.0]
```
Comment (u11d-michal-miler) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]:
```suggestion
 - Serverless framework: [e.g. 3.26.0]
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE]"
+labels: feature
+assignees: u11d-michal-miler, u11d-bartlomiej-galezowski
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (u11d-michal-miler) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]: Let's be more positive :) 

```suggestion
A clear and concise description of what the problem is. Ex.: The plugin helps me a lot in development but it'd be even more helpful if ... [...]
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE]"
+labels: feature
+assignees: u11d-michal-miler, u11d-bartlomiej-galezowski
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```
Comment (u11d-michal-miler) [inline .github/ISSUE_TEMPLATE/feature_request.md:9]: I'd change verbs in title to nouns. What do you think?

```suggestion
**Problem description**
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
```
Comment (u11d-michal-miler) [inline .github/ISSUE_TEMPLATE/feature_request.md:12]:
```suggestion
**Expected solution**
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
```
Comment (u11d-michal-miler) [inline .github/ISSUE_TEMPLATE/feature_request.md:15]:
```suggestion
**Alternatives description**
```

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 7 above)
Comment (u11d-bartlomiej-galezowski) [reply .github/ISSUE_TEMPLATE/feature_request.md:9]: Yes, that is more readable.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (u11d-bartlomiej-galezowski) [reply .github/ISSUE_TEMPLATE/bug_report.md:16]: 1. Good idea.
2. Yes, we should do this. I will create an issue when templates will merged.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+
+**Bug description**
+A clear and concise description of what the bug is.
+
+**Steps to reproduce**
+Steps to reproduce the behavior:
+1. 
+
+**Current behavior**
+A clear and concise description of current behavior.
```
Comment (u11d-michal-miler) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]:
```suggestion
A clear and concise description of the current behavior.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+labels: bug
+assignees: u11d-bartlomiej-galezowski, u11d-michal-miler
+
+---
+
+**Bug description**
+A clear and concise description of what the bug is.
+
+**Steps to reproduce**
+Steps to reproduce the behavior:
```
Comment (u11d-michal-miler) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]:
```suggestion
Steps to reproduce the bug:
```

**Review by u11d-michal-miler — APPROVED** [review]
Looks good to me. Two minor comments added. Prettier will be configured in the upcoming PR.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 11 above)
Comment (u11d-bartlomiej-galezowski) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I've changed both lines.

**Review by u11d-michal-miler — APPROVED** [review]

**Pull request merged**